workflows/eager.nf

```diff
@@ -365,6 +370,18 @@ workflow EAGER {
         ch_multiqc_files = ch_multiqc_files.mix( ESTIMATE_CONTAMINATION.out.mqc.collect{it[1]}.ifEmpty([]) )
     }
 
+    //
+    // SUBWORKFLOW: aDNA Damage Manipulation
+    //
+
+    if ( params.run_mapdamage_rescaling || params.run_pmd_filtering || params.run_trim_bam ) {
+        MANIPULATE_DAMAGE( ch_dedupped_bams, ch_fasta_for_deduplication.fasta )
+        ch_versions            = ch_versions.mix( MANIPULATE_DAMAGE.out.versions )
```

Comment:
Missing MultiQC file mixing (this needs to go at the bottom of the `eager.nf`)

Reply:
Added